plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -1272,6 +1279,64 @@ private void testDeletionVectorsDisabledCreateTable(String tableDefinition)
         }
     }
 
+    @Test
+    void testDeletionVectorsEnabledWriteCheckpoint()
+            throws Exception
+    {
+        testDeletionVectorsEnabledWriteCheckpoint("(x int) WITH (deletion_vectors_enabled = true, checkpoint_interval = 2)");
+        testDeletionVectorsEnabledWriteCheckpoint("WITH (deletion_vectors_enabled = true, checkpoint_interval = 2) AS SELECT 1 x");
+    }
+
+    private void testDeletionVectorsEnabledWriteCheckpoint(String tableDefinition)
+            throws Exception
+    {
+        try (TestTable table = newTrinoTable("deletion_vectors", tableDefinition)) {
+            assertThat((String) computeScalar("SHOW CREATE TABLE " + table.getName()))
+                    .contains("deletion_vectors_enabled = true");
```

Comment:
Let's remove unhelpful assertions. The following `deletion_vector_.*.bin` check is sufficient in my opinion.